### DISCUSSION TITLE
Hamburger nav menu shows only sibling pages with no way to ascend navigation

### DISCUSSION THREAD

**Discussion by reece** [squidfunk/mkdocs-material#6350] [General]
Code for the following question is at https://github.com/HGVSnomenclature/hgvs-nomenclature .

When a window is narrow (or on mobile) the navigation menu shows only sibling links. There is no way to navigate up. 

For example, on https://hgvs-nomenclature.org/en/latest/recommendations/DNA/substitution/, I see this on a wide window:
![image](https://github.com/squidfunk/mkdocs-material/assets/109453/8b5fbd92-91bd-43ac-b31c-ca8002b6b75f)

But, on a narrow window, I see
![image](https://github.com/squidfunk/mkdocs-material/assets/109453/a6cbe988-c854-4a59-bbfd-76c142d36ed4)

There appears to be no way to ascend to the parent navigation items.

Is this a bug or misconfiguration on my part?

**Comment by alexvoss** [comment]
This should work and you can see the intended behaviour on the [documentation pages](https://squidfunk.github.io/mkdocs-material/), which work fine with a whole range of display widths.  It would help us if you could produce a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) that shows the problem with a minimal `mkdocs.yml` and shows us exactly what versions you are using. In the process, you may already identify which combination of features and plugins might be to blame.

**Reply by reece** [reply]
My fault. I had a stray css selector that set that element to display: none. It was unfortunately working exactly as I had directed. Thanks for the reply. Closing.
